joeflow/runner/dramatiq.py

```diff
@@ -56,10 +56,10 @@ def _dramatiq_task_runner(task_pk, workflow_pk, retries=0):
             logger.exception("Execution of %r failed", task)
         else:
             if result is False:
-                logger.info("Task returned False, retrying …")
-                raise RetryError("Task returned False, retrying …")
+                logger.info("%r returned False, retrying …", task)
+                raise RetryError("%r returned False, retrying …" % task)
```

Comment:
As much as I appreciate the logger change, this might prevent tools like Sentry from doing proper error stacking. Since we have the info log, I don't think we need the task again as part of the exception message.

Reply:
reverted